checks.d/vsphere.py

```diff
@@ -582,6 +589,18 @@ def _cache_morlist_raw_atomic(self, i_key, obj_type, obj, tags, regexes=None, in
                     args=(i_key, 'host', host, tags_copy, regexes, include_only_marked)
                 )
 
+        elif obj_type == 'folder':
```

Comment:
Can we have nested folders? If those are possible, then we wouldn't recurse into them with the current logic. Also, can we only have folders in the `datacenter` level? Or can other resources be arranged into folders as well.